controllers/process.go

```diff
@@ -255,16 +256,20 @@ func (r *ServicefenceReconciler) refreshFenceStatusOfService(ctx context.Context
 				},
 				Spec: lazyloadv1alpha1.ServiceFenceSpec{
 					Enable: true,
+					WorkloadSelector: &lazyloadv1alpha1.WorkloadSelector{
```

Comment:
later we may add a config option to allow customizeing this like use selector from service for auto-created servicefence.